test/unit/TestNfa.ml

```diff
@@ -0,0 +1,162 @@
+open Lib
+open Bits
+module Map = Nfa.Map
+
+let ( let* ) = Option.bind
+
```

Comment:
It should be in the utils.

Reply:
Should it really? 🤔
`let*` could be used as symlink to bind function for any monad, not just Option. 
If you think it's unlikely we will use other monads in the future, I can move it to Utils

Reply:
Ah, missed this point. I'm sorry.